core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -47,6 +48,15 @@ private[spark] class Pool(
   var name = poolName
   var parent: Pool = null
 
+  override def maxRunningTasks: Int = {
```

Comment:
Does it make sense to move this to Schedulable.scala? It looks like Pool and TaskSetManager both have the same implementation (assuming that Int.MAX_VALUE is the default).